\input{header}
%lung, colon, and stomach?

\begin{document}
% \section{Title}
% \maketitle

\section{Project Summary/Abstract} % 30 lines of text

\input{abstract}

\clearpage

\section{Specific Aims} %1 page

\input{aims}

\clearpage

\section{Research Strategy} %6 pages

% 6 pages
% Significance : Recommended Length: Approximately 0.5-1 pages 
% Innovation : Recommended Length: The recommended length of the innovation section is 0.25-0.5
% Approach : Recommended Length: The maximum recommended length of the approach section is 4.5-5 pages. 

\section{Significance}

% why study somatic mutation?
Somatic mutations are mutations that occur in somatic cells, and as such are not passed to offspring.
These mutations arise due to errors in DNA replication preceding cell division.
While the DNA replication and error repair machinery is highly accurate, mutations occur at high enough frequencies to be significant.
Current assumptions of error rate and genome size imply that on average each cell division results in the accumulation of one novel mutation in the daughter cells.

These mutations are central to the somatic mutation theory of cancer.
This theory posits that cancers arise in multicellular organisms when somatic cells of that organism accumulate a sufficient number of mutations in oncogenes or tumor suppressor genes induces tumorigenesis.
Important mutations in oncogenes cause them to be over activated, promoting unnecessary cell proliferation.
In contrast, relevant mutations in tumor suppressor genes cause them to lose functionality, inhibiting their ability to limit cell proliferation.
Once a sufficient number of these mutations accumulate, the cell divides rapidly and accumulates even more somatic mutations, severely destabilizing the genome.
Thus, understanding how somatic mutations accumulate in healthy cells is vital to understanding the onset and progression of cancer. 

%why study chromatin accessibility?
A similarly important factor required for the onset of cancer is the deregulation of chromatin.
The chromatin state in a region of the genome is determined by the organization of the nucleosomes in the region.
These nucleosomes are composed of histones, which bind and organize a segment of DNA.
While the function of higher orders of organization is not well understood, the function and impact of different chromatin states is clear.
Chromatin takes two important possible states: heterochromatin, in which the DNA is tightly packaged; and euchromatin, in which the DNA is loosely packaged.
In the heterochromatin state, translation is repressed as the DNA is inaccessible to translation proteins.
In the euchromatin state, translation can occur as the DNA is available for translation complexes to bind.

As the chromatin state significantly impacts translational ability, and different tissues must express different genes to maintain their identity, different tissues have different distributions of chromatin states.
The process of switching between these states is known as chromatin remodeling.
As cancers require normally silenced genes to be expressed, misregulated chromatin remodeling is important for tumorigenesis.
In fact, many genes implicated as oncogenes or tumor suppressor genes have functions important for chromatin remodeling.
As such, understanding how these epigenetic changes cause disease is vital to understanding and preventing cancer.


% why study the interface of somatic mutation and chromatin accessibility?
Thus, it is apparent that a comprehensive understanding of cancer onset requires integrating both somatic mutation and epigenetic misregulation.
The somatic mutation theory of cancer can explain both components, as mutations in chromatin remodeling proteins can cause the large-scale changes in expression required for tumors to form and thrive.
Some critics claim that the theory inadequately explains how the requisite mutations are obtained.
As somatic mutations occur at very low frequency, the probability of accumulating the number of required mutations in the correct sites and within the same cell is low.
A potential resolution to this inconsistency is the observation that mutation rates are not evenly distributed across the genome.
That is, mutation rates could be high enough in regions that contain oncogenes and tumor suppressor genes that mutating a sufficient number of them can occur with probabilities consistent with rates of cancer incidence.

Recently, many groups have reported a strong association between regional variation in somatic mutation rates in tumors and chromatin states in that region.
This pattern has also been seen in germline mutations.
Thus, it is highly likely that chromatin states impact local mutation rates in healthy somatic tissues as well.
Because chromatin remodeling is an important aspect of cell differentiation, different tissues have different patterns of chromatin states.
This implies that different tissues have different distributions of mutation rates.

One observation that indicates that this difference may be significant for studying cancer is that different tissues have different rates of cancer incidence.
There is debate about how much of this difference can be attributed to difference in cell division rates between different tissues.
However, differences in chromatin states of oncogenes and tumor suppressor genes in healthy somatic tissue may play an important role.

% Aim 1 of this project is to develop a novel bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome in somatic samples.

% why do we need a method to jointly detect somatic mutations and chromatin accessibility?
Despite the central role of somatic mutations in cancer development, they remain difficult to study.
However, if a relationship between chromatin accessibility and mutation rate exists, this can be explicitly modeled to improve detection of somatic mutations.
At the same time, sequencing-based assays of chromatin accessibility hold underlying genotype information.
Thus, considering both types of data at the same time and integrating information from both data sources into a model will increase the ability of the researcher to sensitively and accurately detect somatic mutations.

With this ability, researchers will be able to study somatic mutation in healthy tissue much more easily than previously.
Because the sequencing reads provided by the accessibility assay can be used to estimate genotypes in low-depth regions, researchers can achieve higher-quality genotypes with less sequencing depth.
This could result in lower sequencing costs, which can be an issue when studying somatic mutations due to the high sequencing depth required with traditional methods.
Additionally, this method may be a useful alternative or supplement to current methods used to detect somatic mutations within tumors.

% how will this method change the field if it is achieved?
% reduce sequencing costs - less covg necessary initially, doesn't throw away  information from ATAC-seq
% 

%%%%%%%%%%%%% aim 2 %%%%%%%%%%%%%%%%

% Aim 2 of this project is to test the hypothesis that chromatin accessibility significantly impacts mutation rate.

% why study the interaction of somatic mutation and chromatin accessibility? 

% how will this aim change the field if it is achieved?

% 1. Significance
% Explain the importance of the problem or critical barrier to progress that the proposed project addresses.
% Explain how the proposed project will improve scientific knowledge, technical capability, and/or clinical practice in one or more broad fields.
% Describe how the concepts, methods, technologies, treatments, services, or preventative interventions that drive this field will be changed if the proposed aims are achieved.

\section{Innovation}
% Explain how the application challenges and seeks to shift current research or clinical practice paradigms.
% Describe any novel theoretical concepts, approaches or methodologies, instrumentation or interventions to be developed or used, and any advantage over existing methodologies, instrumentation, or interventions.
% Explain any refinements, improvements, or new applications of theoretical concepts, approaches or methodologies, instrumentation, or interventions.

While there has been a large body of work on the generation of somatic mutations in the presence of mutagens, much less is known about the endogenous process that generates somatic mutations--that is, the process that generates somatic mutations even in cells that are never exposed to mutagens or unusual stress.
% Despite the critical role somatic mutations play in tumorigenesis, these mutations are usually studied only in tumors, as there are a sufficient number of mutations that a significant number can be detected at modest sequencing depth.
% However, most of these mutations did not cause the tumor, and occur only after the tumor has formed.
This impedes researchers' ability to detect significant departures from healthy levels and patterns of somatic mutation; a significant amount of energy is expended in differentiating non-pathogenic "passenger" mutations from the "driver" mutations that actually contribute to tumor formation.
This knowledge gap is largely due to the technical difficulty of detecting somatic mutations in healthy tissues.
In any sample of healthy tissue, the number of somatic mutations within the sample is expected to be orders of magnitude smaller than the number of sequencing errors.
A reliable method of detecting somatic mutations in healthy cells and non-tumor samples will enable researchers to study somatic mutations in healthy tissue, improving their ability to distinguish between natural patterns of somatic variation and unhealthy levels of somatic variation indicative of disease.

% what's new about considering somatic mutations along with chromatin accessibility?

% - healthy somatic tissue rather than germline or cancer tumor
% - design allows detection of irregularities due to increased cancer risk
% - show how a proactive approach to predicting cancer vulnerability could be used for screening

\section{Approach}
\subsection{Aim 1}
\subsubsection{Pipeline Inputs}
The low frequency of mutations in somatic cells can make them difficult to detect.
At the same time, errors introduced while sequencing occur at far greater frequency.
Great care must be taken when using software tools for detecting somatic mutations, as sequencing errors can cause high false positive rates.

A significant number of mutations are expected to occur in microsatellites.
These regions are some of the most variable in the genome, and have been demonstrated to mutate frequently even in somatic cells.
Short sequencing reads are often difficult to map to these repeat-rich regions, so a bioinformatic method to detect mutations in these regions should incorporate long sequencing reads. At the same time, long sequencing reads are particularly error prone, causing them to make more errors when reporting the nucleotide within the sequencing read. Thus, the pipeline should be able to accomodate short sequencing reads as a method for accurately detecting point mutations and long sequencing reads for accurately detecting insertions and deletions.

For the long read platform, Oxford Nanopore is preferred because the electrical signal used to sequence the input DNA also contains information that can be used to detect methylated cytosines without additional library preparation. This information will be used to validate nucleosome occupancy information. That this information can be extracted from the same sequencing run is significant, as the the amount of sampled tissue will be too small to perform a separate bisulfite sequencing reaction.

Oxford Nanopore sequencing can be done in one of two ways: 1D sequencing and 2D sequencing.
In 2D Oxford Nanopore sequencing, each complementary strand of DNA is connected by a hairpin loop. As the DNA is pulled through the pore, the hairpin loop is computationally recognized by the sequencer. It then knows that the sequence following is from the complementary strand of DNA, and uses that information to correct potential errors before outputting the bases contained in the read.
Therefore, 2D Oxford Nanopore reads should be used because they contain fewer errors than 1D reads.

Thus, the pipeline will accept as inputs: 1) a set of short Illumina genome sequencing reads, 2) a set of Oxford Nanopore 2D genome sequencing reads, 3) a set of Illumina sequencing reads generated by an ATAC-seq protocol, and 4) the reference of the genome to which the reads will be aligned. The Illumina reads should be in FASTQ format, the Oxford Nanopore reads should be in FAST5 format, and the reference genome should be in FASTA format.

\subsubsection{Read Correction}
While 2D Nanopore data contains fewer errors than 1D data, it remains far more error prone than short read data.
And while short reads have relatively few errors, they do contain errors that can cause ambiguity during variant detection.
To minimize the number of false positive mutation calls, the first step of the pipeline will be error correcting short reads with the Rcorrector package.

Rcorrector is an Illumina read corrector designed for use in RNA-seq experiments. Though it was designed for RNA sequencing reads, it performs as well as alternatives on whole genome sequencing reads and has improved accuracy in regions with low coverage. Since we will use the pipeline on reads generated with low source DNA, potentially causing highly variable levels of coverage, Rcorrector is the best program for this step.

The Rcorrector parameter that has the most impact on the analysis is the kmer size used to detect and repair errors. To identify the best value of \textit{k} to use, the reads will be corrected using each odd value from 15 to 31. This range was chosen to be centered on the default value of 23 and have an equal number of values on each side of 23 without going over the maximum value the program supports, 32. Each set of corrected reads will be aligned to the reference genome with the BWA software and the set of corrected reads that produced the alignment with the fewest mismatches will be used for downstream analyses.





\subsection{Aim 2}
% 2. Approach
% Describe the overall strategy, methodology, and analyses to be used to accomplish the specific aims of the project. Unless addressed separately in the Resource Sharing Plan attachment, include how the data will be collected, analyzed, and interpreted as well as any resource sharing plans as appropriate.
% Discuss potential problems, alternative strategies, and benchmarks for success anticipated to achieve the aims.
% If the project is in the early stages of development, describe any strategy to establish feasibility, and address the management of any high risk aspects of the proposed work.
% Point out any procedures, situations, or materials that may be hazardous to personnel and the precautions to be exercised. A full discussion on the use of select agents should appear in the Select Agent Research attachment below.

% If you have multiple Specific Aims, you may address Significance, Innovation, and Approach either for each Specific Aim individually or for all of the Specific Aims collectively. As applicable, also include the following information as part of the Research Strategy, keeping within the three sections (Significance, Innovation, and Approach) listed above.

\subsubsection{Power Analysis}

The distribution of accessibility values for each mutation is not known. However, assuming the distribution is Poisson can assist in approximating an appropriate sample size for the experiment. Additionally, assume an average coverage of 5X for the coverage of a mutated site. The square root of a Poisson distribution is approximately normally distributed with mean $\sqrt{\lambda}$ and standard deviation 0.25.
By Lehr's equation, to distinguish between two rates $\lambda_1$ and $\lambda_2$ with power $\beta$ and significance $\alpha$, the required number of observations is $\frac{2 * (Z_{1 - \frac{\alpha}{2}} + Z_{1 - \beta})^2}{{\sqrt{\lambda_1} - \sqrt{\lambda_2}}^2}$. With $\alpha$ = 0.05 and $\beta$ = 0.2, a 5\% increase in the rate parameter from 5 to 5.25 can be detected using more than 1312 mutations.

Though base-pair level accuracies of 99.78\% have been achieved using mixed Illumina and Nanopore reads, assume only 50\% of all mutations are detected, and these are evenly distributed across all sample sources and relative chromatin accessibility. Literature estimates \parencite{behjati_genome_2014} of similar somatic mouse samples suggest greater than 160 base substitutions per sample, resulting in 80 detectable substitutions. Microsatellite insertions and deletions should occur at equal or greater frequencies, but to be conservative those will not be considered. Under these assumptions, the required number of mice to sample is 16.4. To ensure a sufficient number of somatic mutations are acquired, samples will be taken from 30 mice.

\section{References} %no limit

\end{document}