\input{header}
\usepackage{outline}

% \item bar \parencite{zhang_extra_2011}

\begin{document}

% • What do you intend to do?
% • Why is this worth doing or the significance of the research? How is it innovative?
% • What has already been done in general, and what have other researchers done
% in this field? Use appropriate references. What will this new work add to the field
% of knowledge?
% • What have you (and your collaborators) done to establish the feasibility of what
% you are proposing to do?
% • How will the research be accomplished? Who? What? When? Where? Why?

% 1. Make sure that all sections are internally consistent and that they dovetail with each
% other. Use a numbering system, and make sections easy to find. Lead the reviewers
% through your research plan. One person should revise and edit the final draft.
% 2. Show knowledge of recent literature and explain how the proposed research will
% further what is already known.
% 3. Emphasize how some combination of a novel hypothesis, important preliminary
% data, a new experimental system and/or a new experimental approach will enable
% important progress to be made.
% 4. Establish credibility of the proposed principal investigator and the collaborating
% researchers. 

% 12 pages
% Significance : Recommended Length: Approximately 1-2 pages 
% Innovation : Recommended Length: The recommended length of the innovation section is 1/2-1
% Approach : Recommended Length: The maximum recommended length of the approach section is 9-10 pages. 

% 6 pages
% Significance : Recommended Length: Approximately 0.5-1 pages 
% Innovation : Recommended Length: The recommended length of the innovation section is 0.25-0.5
% Approach : Recommended Length: The maximum recommended length of the approach section is 4.5-5 pages. 

% Aim 1 - develop a novel bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome
% Aim 2 - test the hypothesis that chromatin accessibility significantly impacts mutation rate
\begin{outline}
	\item Significance
	\begin{outline}
		\item Somatic mutations play an important role in carcinogenesis
		\begin{outline}
			\item Genome instability is a hallmark of cancer \parencite{hanahan_hallmarks_2000, hanahan_hallmarks_2011}
			\item The Somatic Mutation Theory of cancer suggests that somatic mutations slowly accumulate until a sufficient number disrupt normal cell cycle control \parencite{tomasetti_variation_2015}
			\item There is disagreement on the role somatic mutations play in carcinogenesis \parencite{baker_cancer_2015}
			\begin{outline}
				\item The Somatic Mutation Theory doesn't explain how a sufficient number of somatic mutations accumulate \parencite{baker_cancer_2015}
				\item Variation in somatic tissue division rates explains a large proportion of variation in incidence by tissue \parencite{tomasetti_variation_2015} implying cancer is caused by mutations that occur during cell division, but there is disagreement \parencite{rozhok_critical_2015, wang_implications_2015}
				\item Our understanding of the rate of somatic mutations and how that rate changes in different types of cells is limited \parencite{stratton_cancer_2009}
			\end{outline}
		\end{outline}
		\item The role of chromatin states in somatic mutation
		\begin{outline}
			\item Analysis of neutrally-evolving regions suggests that several measurements of epigenetic states are associated with variation in local mutation rate in the germline \parencite{makova_effects_2015}
			\item Hi-C analysis combined with cancer genome data has suggested high-level chromatin organization contributes to simultaneous point mutations in protein-coding genes \parencite{shi_chromatin_2016}
			\item A single histone modification H3K9me3 can account for nearly 40\% of variation in regional point mutation rates in cancer cells \parencite{schuster-bockler_chromatin_2012}
			\item Cytosine to Thymine mutations caused by deamination at mehtylated CpG sites is associated with common driver mutations in colorectal and ovarian cancers \parencite{gold_somatic_2017}
		\end{outline}
		\item Measuring chromatin states
		\begin{outline}
			\item Sequence Characteristics \parencite{makova_effects_2015}
			\begin{outline}
				\item GC content
				\item Sex chromosome vs. autosome
				\item Exon density
				\item Recombination rate
				\item Distance to telomere
				\item Microsatellite density
				\item Density of histone modifications
			\end{outline}
			\item Mutation Types Associated with Chromatin States
			\begin{outline}
				\item foo
			\end{outline}
			\item Next-Gen Sequencing Assays
			\begin{outline}
				\item DNase-seq
				\item ATAC-seq
			\end{outline}
		\end{outline}
	\end{outline}
	\item Innovation
	\begin{outline}
		\item Base-pair resolution of actual chromatin accessibility - not proxy of accessibility
		\item Sampling strategy allows analysis in healthy somatic cells rather than germline or cancer cells, which could be different.
		\item Combining long and short read technologies allows accurate detection of single-nucleotide, insertion, and deletion mutations
	\end{outline}
	\item Approach
	\begin{outline}
		\item Aim 1 - develop a novel bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome
			\begin{outline}
				\item Short Read Error Correction
				\item Long Read Error Correction
				\item Error correct ATAC-seq reads with sufficient length
				\item Align
				\item Call Variants
				\item Estimate Chromatin Accessibility
				\item Alternative Approaches
			\end{outline}
		\item Aim 2 - test the hypothesis that chromatin accessibility significantly impacts mutation rate
			\begin{outline}
				\item Sample Sources
				\item Tissue collection
				\item Sequencing
				\item ATAC-seq
				\item Statistical Analysis of Results
				\item Alternative Approaches
				\item Data
			\end{outline}
	\end{outline}
\end{outline}
\medskip

\printbibliography






\end{document}







