\documentclass{beamer}
\usepackage{tikz}
\usepackage{graphicx}
\usepackage[style=nature, citestyle=authoryear]{biblatex}
\usetikzlibrary{shapes,arrows,calc,positioning}
\bibliography{comps}

\usefonttheme{professionalfonts}
\usetheme{Boadilla}
\setbeamertemplate{navigation symbols}{}%remove navigation symbols
\hypersetup{pdfstartview={Fit}} % fits the presentation to the window when first displayed
\graphicspath{{./figures/}{../written/}}
\emergencystretch=1em

%Info
\title[Chromatin and Mutation]{Evaluating the Relationship between Chromatin State and Mutation Rate in Cancer}
\titlegraphic{\includegraphics[width=.3\linewidth]{figure.pdf}}
\date{11/9/17}
\author{Adam Orr}

\begin{document}

\frame{\titlepage}

\begin{frame}{Cancer is a major health problem}
\begin{columns}
\column{.5\textwidth}
\begin{itemize}
\item Estimated 1.6 million new cancer cases and 600,000 deaths in 2016 \footnotemark
\item Cancer is difficult to treat
\item Many cancers acquire drug resistance \parencite{holohan_cancer_2013}
\end{itemize}
\column{.5\textwidth}
\includegraphics[width=.\linewidth]{}
\end{columns}
\footnotetext[1]{\url{https://www.cancer.gov/about-cancer/understanding/statistics}}
\end{frame}

\begin{frame}{Cancer is believed to be caused by somatic mutations}
\end{frame}

\begin{frame}{Knowing how cancer starts is critical for prevention}
\end{frame}

\begin{frame}{RViMR may explain how so many mutations occur in the correct places}
\end{frame}

\begin{frame}{Epigenetic markers significantly correlate with local mutation rate}
\end{frame}

\begin{frame}{Nucleosome position and occupancy affect local mutation rate}
\end{frame}

%how do we study nucleosomes?

\begin{frame}{DNAse-seq}
\end{frame}

\begin{frame}{ChIP-seq}
\end{frame}

\begin{frame}{Transposase}
\end{frame}

\begin{frame}{ATAC-seq}
\end{frame}

%how do we find mutations?

\begin{frame}{Illumina Sequencing is good for detecting SNPs}
\end{frame}

\begin{frame}{Nanopore Sequencing is good for detecting insertions and deletions}
\end{frame}

%what do we need to consider for somatic samples?

\begin{frame}{Heterogeneity is a problem}
\end{frame}

\begin{frame}{Single cell technology a potential solution, but comes with many more problems}
\end{frame}

\begin{frame}{Sequencing errors are a problem}
\end{frame}

\begin{frame}{Reduce number of PCR steps to reduce errors in library prep}
\end{frame}

\begin{frame}{Read correction reduces errors}
\end{frame}

\begin{frame}{Small amount of source DNA is a problem}
\end{frame}

%with all this in mind, our aims!

\begin{frame}{How does chromatin accessibility correlate with mutation rate in the same tissue? How does it change in cancer?}
\end{frame}

%pipeline stuff
%for the pipeline, do left half text, right half highlight the part of the pipeline we're talking about

\begin{frame}{Pipeline Overview}
\end{frame}

\begin{frame}{Pipeline Validation}
\end{frame}

\begin{frame}{Short Read Correction - Rcorrector}
\end{frame}

\begin{frame}{Long Read Correction - Nanocorr}
\end{frame}

\begin{frame}{Alignment - Minimap2}
\end{frame}

\begin{frame}{Detect Mutations - MuTect2}
\end{frame}

\begin{frame}{Detect Mutations - Nanopolish}
\end{frame}

\begin{frame}{Calculate Chromatin Accessibility - MACS2}
\end{frame}

\begin{frame}{Tissue Collection}
\end{frame}

\begin{frame}{Library Construction}
\end{frame}

\begin{frame}{Statistical Analysis}
\end{frame}

\begin{frame}{Expected Results - Mutation Rates in Open and Closed Chromatin}
\end{frame}

\begin{frame}{Expected Results - Distribution of Chromatin State at Mutated Sites}
\end{frame}

\begin{frame}{Expected Results - Logistic Regression}
\end{frame}

\begin{frame}{Conclusion}
\end{frame}

\begin{frame}[t, allowframebreaks]
\frametitle{References}
\printbibliography
\end{frame}

\end{document}