\documentclass{beamer}
\usepackage{tikz}
\usepackage{graphicx}
\usepackage[style=nature, citestyle=authoryear]{biblatex}
\usetikzlibrary{shapes,arrows,calc,positioning}
\bibliography{comps}

\usefonttheme{professionalfonts}
\usetheme{Boadilla}
\setbeamertemplate{navigation symbols}{}%remove navigation symbols
\hypersetup{pdfstartview={Fit}} % fits the presentation to the window when first displayed
\graphicspath{{./figures/}{../written/}}
\emergencystretch=1em

%Info
\title[Chromatin and Mutation]{Evaluating the Relationship between Chromatin State and Mutation Rate in Cancer}
\titlegraphic{\includegraphics[width=.3\linewidth]{logistic_regression.pdf}}
\date{11/9/17}
\author{Adam Orr}

\begin{document}

\frame{\titlepage}

\begin{frame}{Cancer is a major health problem}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
\item Estimated 1.6 million new cancer cases and 600,000 deaths in 2016 \footnotemark
\item Cancer is difficult to treat
\item Many cancers acquire drug resistance \parencite{holohan_cancer_2013}
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth]{hanahan_2011_hallmarks.png}\footnotemark
\end{columns}
\footnotetext[1]{\url{https://www.cancer.gov/about-cancer/understanding/statistics}}
\footnotetext[2]{\cite{hanahan_hallmarks_2011}}
\end{frame}

\begin{frame}{Cancer is believed to be caused by somatic mutations}
\includegraphics[width=\linewidth]{stratton_2009_mutations.jpg}
\footnotetext[1]{\cite{stratton_cancer_2009}}
\end{frame}

\begin{frame}{Understanding cancer initiation is critical for prevention}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
\item Some claim Somatic Mutation Theory doesn't explain how sufficient mutations accumulate \parencite{baker_cancer_2015}
\item Disputed evidence that lifetime cancer incidence is linearly correlated with number of somatic cell divisions \parencite{tomasetti_variation_2015}
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth]{tomasetti_2015_risk.png}
\end{columns}
\footnotetext[1]{\cite{tomasetti_variation_2015}}
\end{frame}

\begin{frame}{RViMR may explain how so many mutations occur in the correct places}
\includegraphics[width=\linewidth,trim={0 13cm 0 0},clip]{makova_2015_rvimr.jpg}
Mutation rates in 1Mb windows across chromosome 1 for different types of mutations \parencite{makova_effects_2015}
\end{frame}

\begin{frame}{Epigenetic markers significantly correlate with local mutation rate}
\includegraphics[width=\linewidth,trim={0 7.5cm 0 0},clip]{polak_2015_chromatin.jpg}
\begin{itemize}
\item Closed chromatin regions correlate with high single nucleotide mutation rates. 
\item Open chromatin regions correlate with high insertion/deletion rates \parencite{makova_effects_2015}
\end{itemize}
% \includegraphics[width=\linewidth]{}
\footnotetext[1]{\cite{polak_cell--origin_2015}}
\end{frame}

% \begin{frame}{Nucleosome position and occupancy affect local mutation rate}
% \begin{columns}
% \column{.4\textwidth}
% \column{.6\textwidth}
% % \includegraphics[width=\linewidth]{}
% \end{columns}
% \end{frame}

%how do we study nucleosomes?

\begin{frame}{DNAse-seq}
\begin{columns}
\column{.4\textwidth}
\column{.6\textwidth}
\includegraphics[width=\linewidth]{song_2010_dnase-seq.png}
\end{columns}
\footnotetext[1]{\cite{song_dnase-seq:_2010}}
\end{frame}

% \begin{frame}{ChIP-seq}
% \begin{columns}
% \column{.4\textwidth}
% \column{.6\textwidth}
% % \includegraphics[width=\linewidth]{}
% \end{columns}
% \end{frame}

\begin{frame}{Transposase}
\begin{columns}
\column{.4\textwidth}
\column{.6\textwidth}
\includegraphics[width=.8\linewidth]{reznikoff_2003_tn5.png}
\end{columns}
\footnotetext[1]{\cite{reznikoff_tn5_2003}}
\end{frame}

\begin{frame}{ATAC-seq}
\includegraphics[width=\linewidth,trim={0 24cm 0 0},clip]{buenrostro_2013_atac.jpg}
\footnotetext[1]{\cite{buenrostro_transposition_2013}}
\end{frame}

%how do we find mutations?

\begin{frame}{Illumina Sequencing is useful for detecting SNPs}
\includegraphics[width=\linewidth,trim={.5in 2.5cm .5in 14cm},clip]{illumina_techspotlight_p2.pdf}
\footnotetext[1]{\url{https://www.illumina.com/documents/products/techspotlights/techspotlight_sequencing.pdf}}
\end{frame}

\begin{frame}{Illumina Sequencing is useful for detecting SNPs}
\centering
\includegraphics[width=.5\linewidth,trim={1in 9cm 4.2in 4in},clip]{illumina_intro_p5.pdf}
\footnotetext[2]{\url{https://www.illumina.com/content/dam/illumina-marketing/documents/products/illumina_sequencing_introduction.pdf}}
\end{frame}



\begin{frame}{Nanopore Sequencing is useful for detecting insertions and deletions}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
\item{A voltage is generated across the membrane containing the nanopore.}
\item{Molecules that pass through the pore cause a detectable shift in current.}
\item{$\alpha$-hemolysin is used as a biological pore}
\item{Helicase is used as a motor}
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth,trim={6cm 0 3cm 0},clip]{nanopore_seq-0.png}
\end{columns}
\footnotetext[1]{\url{https://nanoporetech.com/how-it-works}}
\end{frame}

%what do we need to consider for somatic samples?

\begin{frame}{Heterogeneity is a problem}
\begin{columns}
\column{.4\textwidth}
This can be reduced by careful sampling of tissue

\column{.6\textwidth}
\includegraphics[width=\linewidth]{junttila_2013_heterogeneity.jpg}
\end{columns}
\footnotetext[1]{\cite{junttila_influence_2013}}
\end{frame}

\begin{frame}{Sequencing errors are a problem}
These errors are caused by
\begin{itemize}
\item DNA damage
\item Mutations during library preparation
\item Sequencer measurement error
\end{itemize}

The first two can be minimized by careful DNA extraction methods and minimizing the number of PCR steps required before sequencing.

\vskip 1em

Remaining errors can be corrected \textit{in silico}
\end{frame}

\begin{frame}{Read correction reduces errors}
\begin{columns}
\column{.5\textwidth}
In short reads:
\begin{enumerate}
	\item Create De Bruijn graph and K-mer count table
	\item Determine a global threshold and local threshold
	\item For every read containing a K-mer with a count lower than the threshold, make the minimum number of changes to the kmers to make them higher than the threshold.
\end{enumerate}
\column{.5\textwidth}
In long reads:
\begin{itemize}
	\item The reads are long enough that there is a significant level of overlap between reads. Use this overlap to find a consensus.
	\item \textbf{Or} align short reads to the long reads to "polish" the long reads and repair errors.
\end{itemize}
\end{columns}
\end{frame}

%with all this in mind, our aims!

\begin{frame}{Aims}
How does chromatin accessibility correlate with mutation rate in the same tissue? How does it change in cancer?

\vskip 1em

\textbf{Aim 1}: Develop a bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome in somatic samples.
\begin{itemize}
	\item Use long reads for structural variation, short reads for single nucleotide changes
	\item Error correct reads
\end{itemize}

\vskip 1em

\textbf{Aim 2}: Test the hypothesis that chromatin accessibility significantly impacts mutation rate.
\begin{itemize}
	\item Poisson test of mutation rate in "open" and "closed" regions
	\item Non-parametric test of ATAC-seq enrichment at mutated sites
	\item Logistic regression of ATAC-seq enrichment
\end{itemize}
\end{frame}

%pipeline stuff
%for the pipeline, do left half text, right half highlight the part of the pipeline we're talking about

\begin{frame}{Pipeline Overview}
\centering
\includegraphics[width=.7\linewidth]{figure.pdf}
\end{frame}

\begin{frame}{Pipeline Validation}
%TODO: specify cell lines
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
\item Genotype and ATAC-seq independent intestinal cell lines.
\item Mix the cell lines, then run the pipeline to evaluate effectiveness.
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth]{pipe_validation.pdf}
\end{columns}
\end{frame}

\begin{frame}{Short Read Correction - Rcorrector}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
\item Fix sequencing errors to reduce false positive mutation calls
\item Do this by changing low-abundance kmers to high-abundance kmers
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth,trim={4cm 13cm 2cm 0},clip]{figure.pdf}
\end{columns}
\footnotetext[1]{\cite{song_rcorrector:_2015}}
\end{frame}

\begin{frame}{Long Read Correction - Nanocorr}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
	\item BLAST each short read to long reads
	\item Fix long reads using aligned short reads
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth,trim={4cm 12cm 8cm 0},clip]{figure.pdf}
\end{columns}
\footnotetext[1]{\cite{goodwin_oxford_2015}}
\end{frame}

\begin{frame}{Alignment - Minimap2}
\includegraphics[width=\linewidth,trim={0 9cm 0 4cm},clip]{figure.pdf}
\begin{itemize}
	\item Split the reference into windows, and for a fixed kmer size, rank the kmers and pick the smallest.
	\item This is the \textit{minimizer}
	\item The \textit{minimizer} of a query is used to find the a match in a large reference database, and extended 
\end{itemize}
\footnotetext[1]{\cite{li_minimap2:_2017}}
\end{frame}

% MuTect2 detects mutations by modeling the likelihood of the 3 possible mutations with a uniform error rate and considering any base where the log ratio of the likelihood of a true mutation of that base to no mutation exceeds a threshold. It then estimates the fraction of the sample containing the mutation; if it is sufficiently large, the mutation is emitted.
\begin{frame}{Detect Mutations in Short Reads - MuTect2}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
	\item Ignore sites that are similar to control sample
	\item Calculate likelihood of a mutation for each non-reference base exists at frequency \textit{f}, $L(M^m_f)$
	\item Determine whether $\log_{10} \frac{L(M^m_f)}{L(M_0)}$ exceeds a threshold
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth,trim={6cm 4cm 2cm 7cm},clip]{figure.pdf}
\end{columns}
\footnotetext[1]{\cite{cibulskis_sensitive_2013}}
\end{frame}

\begin{frame}{Detect Mutations in Long Reads - Nanopolish}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
	\item Use electrical signals and a Hidden Markov Model to detect variants
\end{itemize}
\column{.6\textwidth}
\centering
\includegraphics[width=.7\linewidth,trim={2cm 2cm 10cm 5cm},clip]{figure.pdf}
\end{columns}
\footnotetext[1]{\cite{loman_complete_2015}}
\end{frame}

\begin{frame}{Calculate Chromatin Accessibility - MACS2}
\begin{columns}
\column{.4\textwidth}
\begin{itemize}
	\item In a sliding window count:
	\begin{itemize}
		\item ATAC reads
		\item control sequencing reads
	\end{itemize}
	\item A Poisson rate ratio test is performed to see if the count of ATAC reads non-randomly exceeds the count of control reads.
\end{itemize}
\column{.6\textwidth}
\includegraphics[width=\linewidth,trim={0 4cm 8cm 8cm},clip]{figure.pdf}
\end{columns}
\footnotetext[1]{\cite{zhang_model-based_2008}}
\end{frame}

\begin{frame}{Aim 2 - Tissue Collection}
\begin{columns}
\column{.5\linewidth}
\includegraphics[width=\linewidth,trim={0 0 22cm 0},clip]{alonso_2003_skin.jpg}
\column{.5\linewidth}
\includegraphics[width=\linewidth,trim={0 0 4.2cm 0},clip]{casali_2009_intestine.jpg}
\end{columns}
\footnotetext[1]{\cite{alonso_stem_2003}}
\footnotetext[2]{\cite{casali_intestinal_2009}}
\end{frame}

\begin{frame}{Statistical Analysis}
Interesting questions to test for an association and model it for SNPs, insertions, and deletions:
\begin{enumerate}
	\item Is the mutation rate different in "open" and "closed" chromatin regions?
	\item Is the distribution of accessibility enrichment by ATAC-seq different in a cancer-afflicted sample?
	\item Does chromatin accessibility predict mutation probability in a logistic regression?
\end{enumerate}
\end{frame}

\begin{frame}{Expected Results - Mutation Rates in Open and Closed Chromatin}
\begin{columns}
\column{.5\linewidth}
A Poisson rate test can be used to determine if the ratio of the rate parameters is 1.

\vskip 1em

The quantity $P(X_1 | X_1 + X_2 = k)$ is Binomially distributed with $k$ trials and $p$ parameter $\frac{\frac{n_1}{n_2}}{1 + \frac{n_1}{n_2}}$ when the rate ratio is 1.
\column{.5\linewidth}
\includegraphics[width=\linewidth]{mut_rate.pdf}
\end{columns}
\end{frame}

\begin{frame}{Expected Results - Distribution of Chromatin State at Mutated Sites}
\begin{columns}
\column{.5\linewidth}
A 2-sample Anderson-Darling test can evaluate whether two samples were sampled from the same distribution.
\column{.5\linewidth}
\includegraphics[width=\linewidth]{chr_state.pdf}
\end{columns}
\end{frame}

\begin{frame}{Expected Results - Logistic Regression}
\begin{columns}
\column{.5\linewidth}
A logistic regression regresses a binary response variable against a continuous one

\vskip 1em

An analysis of deviance can then be used to determine if the effect is significant.

\column{.5\linewidth}
\includegraphics[width=\linewidth]{logistic_regression.pdf}
\end{columns}
\end{frame}

\begin{frame}{Conclusion}
\begin{itemize}
	\item The pipeline presented here enables analysis of mutation and chromatin accessibility from a single somatic sample
	\item Genotyping and ATAC-seq of single somatic tissue will help elucidate the role of somatic mutation in cancer incidence
\end{itemize}
\end{frame}

\begin{frame}[t, allowframebreaks]
\frametitle{References}
\printbibliography
\end{frame}

\end{document}