\input{header}
%lung, colon, and stomach?

\begin{document}
% \section{Title}
% \maketitle

% \section{Project Summary/Abstract}

% \input{abstract}

% \clearpage

\section{Specific Aims} %1 page

\input{aims}

\clearpage

% 6 pages
% Significance : Recommended Length: Approximately 0.5-1 pages 
% Innovation : Recommended Length: The recommended length of the innovation section is 0.25-0.5
% Approach : Recommended Length: The maximum recommended length of the approach section is 4.5-5 pages. 

\section{Significance}

% why study somatic mutation?
There is significant disagreement on the role somatic mutations play in carcinogenesis \parencite{baker_cancer_2015}.
However, it is clear that cancer tumors exhibit genotypes distinct from their host, and one of the canonical hallmarks of cancer is genome instability \parencite{hanahan_hallmarks_2000, hanahan_hallmarks_2011}.
One of the most popular theories of cancer is the Somatic Mutation Theory, which suggests that somatic mutations slowly accumulate as cells divide, until a sufficient number are gained that significantly disrupt cell cycle control.
While there is evidence that somatic tissue division rates explains a large proportion of variation in cancer incidence \parencite{tomasetti_variation_2015}, implying most cancers are due to stochastic mutations that occur during replication and are not related to exposure, this result is strongly disputed \parencite{rozhok_critical_2015, wang_implications_2015}. In particular, critics claim that the theory doesn't adequately explain how a sufficient number of somatic mutations accumulate \parencite{baker_cancer_2015}. 
Evidence of somatic mutation rate variation in different tissues and in different locations across the genome could help resolve this dispute.
However, our understanding of how the rate of somatic mutations and how that rate changes in different cell types is limited \parencite{stratton_cancer_2009}.
This understanding is critical for creating effective preventative interventions---without a satisfactory explanation of what causes disease and why disease incidence differs, interventions are unable to address this cause.

Differences in epigenetic state have been proposed as an explanation for local mutation rate variation.
Analysis of neutrally-evolving regions suggests that several indicators of epigenetic state are associated with variation in local mutation rate in primate evolution \parencite{makova_effects_2015}.
A screen of histone modifications found that histone modification H3K9me3 can account for nearly 40\% of variation in regional point mutation rates in cancer cells \parencite{schuster-bockler_chromatin_2012}.
Another study using Hi-C analysis of high-level chromatin states has found that high-level chromatin organization contributes to simultaneous point mutations in protein-coding genes \parencite{shi_chromatin_2016}.
Recently, cytosine to thymine mutations caused by deamination of methylated CpG sites has been found to be associated with common driver mutations in colorectal and ovarian cancers \parencite{gold_somatic_2017}.
Thus, it is clear that chromatin states have an impact on local variation in mutation rate; however, more evidence is required to evaluate the strength of this relationship, how it changes in cancer, and how it contributes to differences in mutation and cancer incidence rates in different tissues.

\textbf{This proposal will model the relationship between chromatin state and mutation rate to address this knowledge gap.}
It will create knowledge necessary to explain how differences in chromatin states contribute to cancer, which will aid in the creation of interventions to prevent the disease. Further, it will create an open-source computational resource for others studying cancer, chromatin accessibility, and local mutation rate variation.

%if you can't say how people get cancer, how can you prevent it?

% Despite the important role that somatic mutations play in the onset and progression of cancer, 
% Current assumptions of error rate and genome size imply that on average each cell division results in the accumulation of one novel mutation in the daughter cells.
% These mutations are central to the somatic mutation theory of cancer.
% This theory posits that cancers arise in multicellular organisms when somatic cells of that organism accumulate a sufficient number of mutations in oncogenes or tumor suppressor genes induces tumorigenesis.
% Important mutations in oncogenes cause them to be over activated, promoting unnecessary cell proliferation.
% In contrast, relevant mutations in tumor suppressor genes cause them to lose functionality, inhibiting their ability to limit cell proliferation.
% Once a sufficient number of these mutations accumulate, the cell divides rapidly and accumulates even more somatic mutations, severely destabilizing the genome.
% Thus, understanding how somatic mutations accumulate in healthy cells is vital to understanding the onset and progression of cancer. 

%why study chromatin accessibility?
% A similarly important factor required for the onset of cancer is the deregulation of chromatin.
% The chromatin state in a region of the genome is determined by the organization of the nucleosomes in the region.
% These nucleosomes are composed of histones, which bind and organize a segment of DNA.
% While the function of higher orders of organization is not well understood, the function and impact of different chromatin states is clear.
% Chromatin takes two important possible states: heterochromatin, in which the DNA is tightly packaged; and euchromatin, in which the DNA is loosely packaged.
% In the heterochromatin state, translation is repressed as the DNA is inaccessible to translation proteins.
% In the euchromatin state, translation can occur as the DNA is available for translation complexes to bind.

% As the chromatin state significantly impacts translational ability, and different tissues must express different genes to maintain their identity, different tissues have different distributions of chromatin states.
% The process of switching between these states is known as chromatin remodeling.
% As cancers require normally silenced genes to be expressed, misregulated chromatin remodeling is important for tumorigenesis.
% In fact, many genes implicated as oncogenes or tumor suppressor genes have functions important for chromatin remodeling.
% As such, understanding how these epigenetic changes cause disease is vital to understanding and preventing cancer.


% why study the interface of somatic mutation and chromatin accessibility?
% Thus, it is apparent that a comprehensive understanding of cancer onset requires integrating both somatic mutation and epigenetic misregulation.
% The somatic mutation theory of cancer can explain both components, as mutations in chromatin remodeling proteins can cause the large-scale changes in expression required for tumors to form and thrive.
% Some critics claim that the theory inadequately explains how the requisite mutations are obtained.
% As somatic mutations occur at very low frequency, the probability of accumulating the number of required mutations in the correct sites and within the same cell is low.
% A potential resolution to this inconsistency is the observation that mutation rates are not evenly distributed across the genome.
% That is, mutation rates could be high enough in regions that contain oncogenes and tumor suppressor genes that mutating a sufficient number of them can occur with probabilities consistent with rates of cancer incidence.

% Recently, many groups have reported a strong association between regional variation in somatic mutation rates in tumors and chromatin states in that region.
% This pattern has also been seen in germline mutations.
% Thus, it is highly likely that chromatin states impact local mutation rates in healthy somatic tissues as well.
% Because chromatin remodeling is an important aspect of cell differentiation, different tissues have different patterns of chromatin states.
% This implies that different tissues have different distributions of mutation rates.

% One observation that indicates that this difference may be significant for studying cancer is that different tissues have different rates of cancer incidence.
% There is debate about how much of this difference can be attributed to difference in cell division rates between different tissues.
% However, differences in chromatin states of oncogenes and tumor suppressor genes in healthy somatic tissue may play an important role.

% Aim 1 of this project is to develop a novel bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome in somatic samples.

% why do we need a method to jointly detect somatic mutations and chromatin accessibility?
% Despite the central role of somatic mutations in cancer development, they remain difficult to study.
% However, if a relationship between chromatin accessibility and mutation rate exists, this can be explicitly modeled to improve detection of somatic mutations.
% At the same time, sequencing-based assays of chromatin accessibility hold underlying genotype information.
% Thus, considering both types of data at the same time and integrating information from both data sources into a model will increase the ability of the researcher to sensitively and accurately detect somatic mutations.

% With this ability, researchers will be able to study somatic mutation in healthy tissue much more easily than previously.
% Because the sequencing reads provided by the accessibility assay can be used to estimate genotypes in low-depth regions, researchers can achieve higher-quality genotypes with less sequencing depth.
% This could result in lower sequencing costs, which can be an issue when studying somatic mutations due to the high sequencing depth required with traditional methods.
% Additionally, this method may be a useful alternative or supplement to current methods used to detect somatic mutations within tumors.

% how will this method change the field if it is achieved?
% reduce sequencing costs - less covg necessary initially, doesn't throw away  information from ATAC-seq
% 

%%%%%%%%%%%%% aim 2 %%%%%%%%%%%%%%%%

% Aim 2 of this project is to test the hypothesis that chromatin accessibility significantly impacts mutation rate.

% why study the interaction of somatic mutation and chromatin accessibility? 

% how will this aim change the field if it is achieved?

% 1. Significance
% Explain the importance of the problem or critical barrier to progress that the proposed project addresses.
% Explain how the proposed project will improve scientific knowledge, technical capability, and/or clinical practice in one or more broad fields.
% Describe how the concepts, methods, technologies, treatments, services, or preventative interventions that drive this field will be changed if the proposed aims are achieved.

\section{Innovation}
% Explain how the application challenges and seeks to shift current research or clinical practice paradigms.
% Describe any novel theoretical concepts, approaches or methodologies, instrumentation or interventions to be developed or used, and any advantage over existing methodologies, instrumentation, or interventions.
% Explain any refinements, improvements, or new applications of theoretical concepts, approaches or methodologies, instrumentation, or interventions.

I propose a hybrid sequencing approach in this study that will have a significantly improved ability to accurately and sensitively detect mutations. Long read technology is relatively new, but this type of sequencing can greatly improve cancer research. Tumors often contain large genomic rearrangements and abnormalities which cannot be detected by short reads, but can with long reads. Additionally, long read sequencers require little input material, an important consideration for sequencing small tumors.

Additionally, measuring chromatin accessibility using tissue of the same sample that is genotyped will improve the ability to discriminate between variation in chromatin state.
Most analyses linking epigenetic factors with mutation rates make use of data generated by different individuals than those that generated the mutations, masking the effects of individual variation in chromatin state. This project is innovative in its integration of mutation and chromatin accessibility assays in a single sample from a single individual.



% While there has been a large body of work on the generation of somatic mutations in the presence of mutagens, much less is known about the endogenous process that generates somatic mutations--that is, the process that generates somatic mutations even in cells that are never exposed to mutagens or unusual stress.

% This impedes researchers' ability to detect significant departures from healthy levels and patterns of somatic mutation; a significant amount of energy is expended in differentiating non-pathogenic "passenger" mutations from the "driver" mutations that actually contribute to tumor formation.
% This knowledge gap is largely due to the technical difficulty of detecting somatic mutations in healthy tissues.
% In any sample of healthy tissue, the number of somatic mutations within the sample is expected to be orders of magnitude smaller than the number of sequencing errors.
% A reliable method of detecting somatic mutations in healthy cells and non-tumor samples will enable researchers to study somatic mutations in healthy tissue, improving their ability to distinguish between natural patterns of somatic variation and unhealthy levels of somatic variation indicative of disease.

% what's new about considering somatic mutations along with chromatin accessibility?

% - healthy somatic tissue rather than germline or cancer tumor
% - design allows detection of irregularities due to increased cancer risk
% - show how a proactive approach to predicting cancer vulnerability could be used for screening

\section{Approach}
The objectives of this proposal are to: 1) create a bioinformatic pipeline capable of detecting somatic mutations and estimating chromatin accessibility, and 2) use that pipeline to study the effect of increasing chromatin accessibility on mutation rate.

\subsection{Aim 1 - Develop a bioinformatic pipeline to detect somatic mutations and estimate chromatin accessibility across the genome in somatic samples.}
\paragraph{Rationale.}
A dedicated pipeline must be developed to integrate mutation detection and chromatin accessibility assay data to produce analysis-ready data. This data can then be used to determine the relationship between chromatin accessibility and mutation rate.
To accurately detect mutations, both long reads and short reads will be analyzed; short reads will be used to detect single nucleotide mutations and long reads will be used to detect insertions and deletions. Error correction of both types of reads will be used to improve accuracy.
See Figure \ref{fig:pipeline} for for a visual summary of the pipeline.

\subsubsection{Pipeline Validation}

\begin{wrapfigure}{r}{0.66\linewidth}
\centering
\includegraphics[width=\linewidth]{figure}
\caption{A visual summary of the pipeline.}
\label{fig:pipeline}
\end{wrapfigure}

To measure the accuracy of the pipeline in genotyping heterogeneous mixtures and observe the effect of performing ATAC-seq on mixtures of cancerous and normal tissue, an in-vitro sequencing experiment will be performed. A normal mouse intestinal epithelium cell line will be obtained, along with a mouse intestinal cancer cell line. A DNA sample from each of these lines will be independently sequenced with both Illumina and Nanopore sequencing to obtain the genotypes of both lines, and ATAC-seq will be performed on both samples to detect open and closed chromatin regions. After determining the genotypes and ATAC-seq profiles of the pure cell lines, cells from both lines will be mixed and the protocol repeated. The pipeline will then be run using this data, treating the purified normal cell line as a control. To evaluate the performance of the variant detection part of the pipeline, output variants can be compared to the genotype of the cancer cell line. Variants found that are not present in the cell line are likely false positives, which can be confirmed by Sanger sequencing. To evaluate the performance of the ATAC-seq enrichment detection, the enriched regions detected by the pipeline can be compared to those found in the pure cell lines.
This process will be replicated to ensure pipeline performance is consistent. 

\subsubsection{Pipeline Input Data} 
A significant number of mutations are expected to occur in microsatellites.
These regions are some of the most variable in the genome, and have been demonstrated to mutate frequently even in somatic cells \parencite{behjati_genome_2014}.
Short sequencing reads are often difficult to map to these repeat-rich regions \parencite{maruvka_analysis_2017}, so the pipeline will incorporate long sequencing reads. At the same time, long sequencing reads are particularly error prone. Thus, the pipeline will use short sequencing reads as a method for accurately detecting single nucleotide mutations and long sequencing reads for accurately detecting insertions and deletions, particularly in microsatellite regions.
For the long read platform, Oxford Nanopore is preferred because of the low DNA input requirements.
Nanopore sequencing will be done using 1D\textsuperscript{2} sequencing libraries, as this type of library has a reduced error rate.

Thus, the pipeline will accept as inputs: 1) a set of short paired end Illumina genome sequencing reads from a somatic tumor and somatic normal sample, 2) a set of Oxford Nanopore 1D\textsuperscript{2} genome sequencing reads from a somatic tumor and somatic normal sample, 3) a set of Illumina sequencing reads generated by an ATAC-seq protocol of the somatic tumor sample, and 4) the reference of the genome to which the reads will be aligned. The Illumina reads will be in FASTQ format, the Oxford Nanopore reads will be in FAST5 format, and the reference genome will be in FASTA format.

\subsubsection{Read Correction}
While short reads have relatively few errors, they do contain errors that can cause ambiguity during variant detection.
To minimize the number of false positive mutation calls, the first step of the pipeline will be error correcting short reads with the Rcorrector package \parencite{song_rcorrector:_2015}.
Rcorrector will be used for this step as it excels compared to alternatives on data with variable coverage, as may be expected in this experiment due to the low amount of input DNA used.

% Rcorrector is an Illumina read corrector designed for use in RNA-seq experiments. Though it was designed for RNA sequencing reads, it performs as well as alternatives on whole genome sequencing reads and has improved accuracy in regions with low coverage. Since we will use the pipeline on reads generated with low source DNA, potentially causing highly variable levels of coverage, Rcorrector is the best program for this step.

%TODO: move to alternatives

The corrected short reads will be used to detect and correct errors in the long reads. To do this, the Nanocorr \parencite{goodwin_oxford_2015} software package will be used.
% This software uses BLAST to align the short reads to each long read and changes bases with discrepancies to those from the short reads.
Nanocorr is used for this step in the pipeline because it uses short reads to correct the long reads, rather than relying on the overlaps between long reads. Since the pipeline uses both types of reads, this type of correction is possible and is more accurate than relying on long reads alone.
% This will be done for both sets of long reads; those from the control sample will be corrected using the short reads from the control sample, and the long reads from the experimental sample will be corrected using the short reads from the experimental sample.

\subsubsection{Alignment}

After correction, the sequencing reads should more accurately reflect the DNA of the sequenced sample. However, they still may contain some errors. Additionally, as some of samples planned to be sequenced in this experiment are tumors and may be significantly diverged from the reference genome, a read aligner capable of aligning noisy, divergent reads will be used.
Minimap2 \parencite{li_minimap2:_2017} is a program designed for this purpose. Additionally, Minimap2 supports alignment with both Illumina and Nanopore reads. As such, it will be used to align the reads to the reference genome.
% Minimap2 has suggested sets of parameters for different types of data. For mapping the Illumina reads, \textit{-x sr} will be used to indicate that short reads will be mapped. For mapping long reads, \textit{-x map-ont} will be used to indicate that long reads from an Oxford Nanopore will be mapped. Minimap does not map both reads at the same time, so the operations will be done independently. For both executions of the program, the \textit{-a} option will be specified to signify that the output should be in SAM format. Additionally, the \textit{--cs} option will be used to add a CIGAR string tag to the SAM output. This option is necessary to properly encode CIGAR information for very long reads due to a defect in the BAM specification.
After this step, two BAM files will be produced for each sample; one with aligned Nanopore reads and one with aligned Illumina reads.

\subsubsection{Variant Detection}

%nanopolish
Due to the length of the Nanopore reads, they are well suited for detecting structural variation. To do so, the Nanopolish \parencite{loman_complete_2015} software package will be used.
% Nanopolish uses the raw electrical signal data produced by the nanopore during sequencing. Thus, prior to calling variants, the \textit{eventalign} command of Nanopolish must be run to produce an alignment of the electrical signals to each of the reads. Then, the \textit{variants} command of Nanopolish can be used.
Nanopolish was chosen to call variants as it is the current standard in the field for variant calling using Oxford Nanopore data.
%  % TODO: move to alternatives
At the conclusion of this step, a VCF containing structural and single nucleotide variants will be produced for each sample.

%mutect2
While long reads are helpful for finding structural variants, they are deficient in accurately identifying single nucleotide variants. As such, these variants will be detected using the short Illumina reads. Since the sequenced samples are solid tissue and may be highly heterogeneous cancer tumors, the MuTect2 \parencite{cibulskis_sensitive_2013} software will be used to call variants.
% MuTect2 detects mutations by modeling the likelihood of the 3 possible mutations with a uniform error rate and considering any base where the log ratio of the likelihood of a true mutation of that base to no mutation exceeds a threshold. It then estimates the fraction of the sample containing the mutation; if it is sufficiently large, the mutation is emitted.
MuTect2 requires preprocessing of the normal and experimental short read BAM alignments.
This must be done using the \textit{MarkDuplicates} command of the Picard package (http://broadinstitute.github.io) followed by using the Genome Analysis Toolkit's \textit{BaseRecalibrator} module \parencite{mckenna_genome_2010}.
% Additionally, MuTect2 expects base quality scores to be recalibrated. Base quality scores reflect sequencing machine's predicted probability of committing an error when it emitted that base. These scores can be biased such that the values no longer represent the true probability of error. However, this can be detected and reverted \textit{in silico} using the Genome Analysis Toolkit's BaseRecalibrator tool and a database of variable sites.
% These sites are ignored, and the covariance between base quality score and estimated error rate is modeled. Each base quality score is then adjusted to the value predicted by the model. The database of variable sites will be taken from the European Variation Archive, as NCBI is removing support for storing non-human variation data. There are no other tunable parameters for BaseRecalibrator.
% At the conclusion of this step there will be two BAM files with optical duplicates marked and recalibrated base quality scores; one containing reads from an experimental sample, and one containing reads from a control sample.
The BAM files are then suitable for use with MuTect2 to detect mutations in the cancer sample. After MuTect2 is run, a single VCF file is generated that contains the somatic mutations detected in the cancer sample.
Variant calling is a computationally intensive process; this will likely be the rate limiting step in the pipeline, and can take up to half a day. However, the program can be run in parallel to reduce computational time.

%combine vcfs
\subsubsection{Variant Processing}
%snps in short reads trusted, sv's in long reads trusted
The variants detected by MuTect2 using the short reads will be combined with those detected by Nanopolish using long reads. As short reads are deficient in detecting large structural variation and long reads are deficient in detecting single nucleotide variation, only single nucleotide changes will be considered from the variants detected by MuTect2. Likewise, only structural variants will be considered from the variants detected using Nanopolish.
As the structural variants called from the somatic normal sample are likely to be germline mutations, structural variants found in both the normal sample and cancer sample will be removed.
Filtering and merging of the variant sets will be performed using the vcftools package \parencite{danecek_variant_2011}.

%chromatin accessibility with NucleoATAC
\subsubsection{Chromatin Accessibility Assay}
ATAC-seq reads will be aligned to the reference genome using Minimap2.
This alignment will then be filtered with samtools \parencite{li_sequence_2009} to remove reads that map to mitochondrial DNA.
To estimate the chromatin accessibility at the locations of each of these detected somatic mutations, the reads from the ATAC-seq assay will be used in combination with the MACS (Model-based Analysis of ChIP-Seq) software package \parencite{zhang_model-based_2008}.
% MACS models read counts with a Poisson distribution and tests whether the depth in a particular region is significantly higher than expected by chance.
The alignment of corrected Illumina reads will be used as an input control for MACS.
This will produce a BED file containing the regions which experienced significant enrichment in read depth.

% MACS requires the input data to be aligned to the genome. A large proportion of ATAC-seq reads align to the mitochondria, as it is nonnuclear DNA and highly accessible to the transposase employed in the assay. Thus, the ATAC-seq reads will be aligned to the genome using Minimap with the \textit{-x sr} option, as described above. This alignment will then be filtered to remove reads mapping to the mitochondrial region using the samtools software package. The accuracy of enrichment values calculated by MACS can be improved by providing a control input. This control will be the aligned genomic short reads created earlier. However, like the alignment generated by the ATAC-seq reads, reads mapping to the mitochondrial genome will be removed.

% To use MACS for ATAC-seq analysis, a few parameters must be set.
% MACS requires the user to set the estimated mappable genome size; as such, \textit{-g mm} will be used to specify that the reads are from a mouse genome. Additionally, since paired reads will be used, the \textit{--nomodel} option will be set to indicate that MACS should not attempt to model the insert size and calculate it from the read pairs. This will produce a BED file containing genomic regions which are significantly enriched for ATAC-seq reads. This suggests that these regions are enriched for open chromatin.

% Finally, the mutations detected earlier in the pipeline must be compared with these regions to determine whether they fall in open or closed chromatin regions. To do so, the \textit{intersect} command of the bedtools software package will be used. Using the \textit{-c} option will output the number of mutations in each region. Running the tool again, but using the \textit{-v} option, will output the mutations that do not overlap any enrichment region.

%MACS gives fold enrichment - poisson

%is there a correlation between in-region and out-of-region mutation counts?

% in significant regions, is there a relationship between fold-enrichment and number of mutations?

% for each mutation, what is the distribution of enrichment values?

%validation


\subsubsection{Anticipated results, potential pitfalls and alternative approaches}

The software used in this pipeline was chosen for quality and speed. With the exception of MarkDuplicates, vcftools, and MACS, each step in the pipeline can be parallelized to reduce the computational time required for the pipeline to be completed. Because of this, it is possible to iteratively improve the configuration of a tool if its performance is unsatisfactory. Once complete, I conservatively expect a full run of the pipeline to take less than a day for each sample.

To confirm the success of each step of the pipeline, the results of each step will be inspected. Sequencing data will be checked with the FASTQC diagnostic tool (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/).
For alignments, the proportion of unmapped reads and the distribution of mapping quality scores will be inspected to detect contamination or poor aligner performance. Poor variant calling performance will be detected by validation of mutations. Most of the time, these deficiencies can be rectified by tuning software parameters. If not, alternative software will be used.

The software parameter that has the most impact on the pipeline is the kmer size used to detect and repair errors with Rcorrector. To identify the best value of \textit{k} to use, the reads from the validation experiment will be corrected using each odd value from 15 to 31. This range was chosen to be centered on the default value of 23 and have an equal number of values on each side of 23 without going over the maximum value the program supports, 32.
Each set of corrected reads will be aligned to the reference genome with Minimap2. The set of corrected reads that produced the alignment with the fewest mismatches will be used for further analyses.

There are many alternative error correction and alignment algorithms to use as potential alternatives. However, as long reads are a relatively new technology, there are few software tools available for variant calling with long read data.
An alternative method to Nanopolish, Poreseq, has recently been introduced \parencite{szalay_novo_2015}. While Nanopolish is regarded as slightly more accurate in most cases, after validation it may be found that Poreseq more accurately detects variants in this particular experiment. In this case, Poreseq will be used in the pipeline instead of Nanopolish.

%tune -k option of rcorrector

%there may be too few mutations. since mutations will be confirmed by sanger sequencing, it may be worthwhile to consider single nucleotide and structural variation from both evidence sources and rely on validation.

\subsection{Aim 2 - Test the hypothesis that chromatin accessibility significantly impacts mutation rate.}
\paragraph{Rationale.}
To determine the relationship between mutation and chromatin accessibility and how that relationship changes in cancer, I will apply the pipeline described above to data generated by sequencing tumors and normal cells. Sequencing tumor tissue increases the number of mutations detected, providing enough data to confidently detect and model the relationship between chromatin accessibility and mutation rate.
As somatic mutations can be difficult to detect even in tumors, these methods aim to reduce technical error during sample processing and sequencing.

\subsubsection{Study Organisms}
Due to nonuniform sequencing depth and relatively high rates of sequencing error compared to mutation rates, even very sensitive methods will not detect every mutation. Therefore, in order to increase the number of mutations to detect a sufficient number for a meaningful analysis, many individuals will be sequenced. As this study aims to investigate how the relationship between chromatin accessibility changes in cancer and normal tissues, one group will have cancer and the other will not.
Additionally, the group of cancer afflicted individuals should reliably have tumors that can be sequenced. To reduce the effects of standing genetic variation, the cancer afflicted individuals should be as close to genetically identical as possible to the control individuals, and individuals of both groups should be as homozygous as possible to aid in mutation detection.

To meet all these requirements, mice will be used as the study species. In particular, C57Bl/6J mice will be used as the normal control group and C57Bl/6J-Apc\textsuperscript{Min}/J mice will be used as the experimental cancer group. These mice are heterozygous for a nonsense mutation in the \textit{Apc} gene that creates a nonfunctional protein product. \textit{Apc} is a tumor suppressor gene. Once a null mutation in the sole functional copy of \textit{Apc} is obtained that causes a loss or reduction in function, the cell harboring the mutation proliferates rapidly. This most often occurs in the intestine, where many polyps form. In the process of this rapid proliferation, one polyp is likely to acquire additional mutations that cause a malignant tumor to form. Both strains of mice are readily available from The Jackson Laboratory.

\subsubsection{Sample Preparation}

Sample heterogeneity is a significant hurdle for accurately detecting mutations and getting reliable ATAC-seq data. When a sample is excessively heterogeneous, mutations in the sample are of too low frequency to detect. During ATAC-seq, a heterogeneous sample will have a mixture of nucleosome-free regions, dampening signals that can be detected in more homogenous samples. At the same time, single-cell sequencing and single-cell ATAC-seq require sorting methods that may damage the cell, and interpretation of results is made difficult by mutations introduced and propagated during library preparation. When mutations or ATAC signals are accurately detected, they may be unique to the isolated cell and not representative of the tissue bulk.

Intestine epithelial cells will be extracted from the mice and used as the sample source to avoid these issues. This is another reason the \textit{Apc}\textsuperscript{Min} mice model works well for this study; the polyps and tumors formed are derived from intestinal epithelium. By using a single cell type, a large degree of heterogeneity can be eliminated. To serve as a normal tissue control, keratinocytes from the mouse's skin will also be isolated and sequenced. Cells from both types of mouse will be isolated in the same manner. Mice will be aged to 8 weeks before euthanasia. At this age, most \textit{Apc}\textsuperscript{Min} mice will have multiple intestinal tumors, and after this age there is not a significant increase in the number of size of the tumors. Mice will be euthanized by CO\textsubscript{2} asphyxiation. The skin will be sampled such that a similar amount of tissue is collected for both the intestine and skin sample.

%isolating skin keratinocytes \parencite{jensen_isolation_2013}
To isolate skin cells, a protocol adapted from \cite{jensen_isolation_2013} will be used.
In a sterile hood, the euthanized mice will be submerged in Betadine to disinfect the skin. The mice will then be washed in sterile water. A section of skin will then be excised and soaked in phosphate buffered saline (PBS) to aid in the removal of fat and muscle with a scalpel. The skin will then be treated with trypsin to reduce cell adhesion. This will allow the epidermis to be separated from the dermis with a scalpel. The epidermal sections will then be 
added to fibroblast growth medium and gently stirred. The cells will then be passed through a 70\mu m filter and centrifuged. The pellet of cells will then be washed with PBS to remove any debris, then resuspended in fibroblast growth medium.

%isolating intestinal cells \parencite{orourke_isolation_2016}
To isolate the intestinal cells, a protocol adapted from \cite{orourke_isolation_2016} will be used.
The intestine will be removed and flushed with PBS. It will then be cut open with scissors and washed again with PBS to ensure there is no remaining debris. The intestine will then be minced and incubated with EDTA to prepare for collagenase digestion. The EDTA will then be removed by aspiration and the cells washed with PBS. The cells will then be incubated with basal medium containing collagenase to dissociate the cells. PBS will then be added, the tube mixed, and the PBS layer transferred to a separate tube containing basal medium. The cells will then be filtered through a 70 \mu m filter into a tube coated with bovine serum albumin to prevent adhesion to the side of the tube. This will then be centrifuged, the supernatant discarded, and the cells resuspended in basal medium.

\subsubsection{Power Analysis}

The distribution of accessibility values for each mutation is not known. However, assuming the distribution is Poisson can assist in approximating an appropriate sample size for the experiment. Additionally, assume an average coverage of 5X for the coverage of a mutated site. The square root of a Poisson distribution is approximately normally distributed with mean $\sqrt{\lambda}$ and standard deviation 0.25.
By Lehr's equation, to distinguish between two rates $\lambda_1$ and $\lambda_2$ with power $1 - \beta$ and significance level $\alpha$, the required number of observations is $\frac{2 * (Z_{1 - \frac{\alpha}{2}} + Z_{1 - \beta})^2}{{\sqrt{\lambda_1} - \sqrt{\lambda_2}}^2}$. With $\alpha$ = 0.05 and $\beta$ = 0.2, a 5\% increase in the rate parameter from 5 to 5.25 can be detected using more than 1312 mutations.

Though base-pair level accuracies of 99.78\% have been achieved using mixed Illumina and Nanopore reads, assume only 50\% of all mutations are detected and these are evenly distributed across all sample sources and relative chromatin accessibility. Literature estimates \parencite{behjati_genome_2014} of similar somatic mouse samples suggest greater than 160 base substitutions per sample, resulting in 80 detectable substitutions. Microsatellite insertions and deletions should occur at equal or greater frequencies, but to be conservative those will not be considered. Under these assumptions, the required number of mice to sample is 16.4. To ensure a sufficient number of somatic mutations are acquired, I will take samples from 50 mice of each strain.


\subsubsection{Sample Sequencing}

Each of the intestinal samples will be split into four portions for sequencing, one portion for each sequencing reaction to be performed: whole genome Illumina sequencing, whole genome Nanopore sequencing, and ATAC-seq. The additional portion will be stored and later sequenced by Sanger sequencing to validate mutations detected by the computational pipeline discussed above. The skin samples will be split into three portions for sequencing; they will be treated identically to the intestinal samples, but an ATAC-seq procedure will not be performed.

%illumina prep
DNA extraction for Illumina sequencing will be done by lysing the cells with detergent and proteinase K followed by DNA extraction with chloroform followed by precipitation with ethanol. RNA will then be removed with treatment by RNAse. Once the DNA is extracted, sample preparation can proceed using the Illumina paired-end, PCR-free library preparation kit.

%nanopore prep 
In nanopore sequencing, care must be taken when extracting DNA so long fragments in the sample are not sheared. As such, cells will be lysed by pulverization with glass beads. DNA can then be purified by the same method used for preparation of the Illumina libraries. After the DNA is purified from the sample, the library can be created using the Oxford Nanopore Technologies PCR-free 1D\textsuperscript{2} library preparation kit. 

%atac prep goes here \parencite{buenrostro_atac-seq:_2015}
ATAC-seq DNA extraction and library preparation will be performed by a protocol adapted from \cite{buenrostro_atac-seq:_2015}.
Care must be taken during extraction of DNA for ATAC-seq, as nucleosomes bound to DNA must be extracted intact for the assay to work properly. For this purpose, the cells are lysed using the nonionic, non-denaturing detergent IGEPAL CA-630. The cells are then pelleted in a centrifuge and resuspended in Illumina transposase reaction buffer with the Illumina Nextera transposase. After incubation, the DNA will be purified with a DNA purification kit. The extracted DNA will then be amplified by PCR; during this process, sequencing adapters and barcodes are added. To reduce the uneven amplification of DNA caused by low reagent concentrations toward the end of the reaction, a small portion of the DNA is subjected to a qPCR reaction to determine the appropriate number of cycles for the PCR.

After the data from each sample is analyzed by the pipeline created in Aim 1, DNA extracted from the last portion of the sample will be sequenced by Sanger sequencing to validate the mutations detected by the pipeline.

\subsubsection{Statistical Analysis}
%TODO: Make sure to mention separating types of mutation (insertion, deletion, snp) for doing regression
%other genomic features may also need to be added to regression

After DNA extraction and sequencing, the data will be analyzed by the pipeline created for Aim 1. The output of the pipeline containing the detected mutations and whether they fall within open or closed chromatin regions will be analyzed using the R statistical software package \parencite{r_stats}.
The number of mutations falling in or out of open chromatin regions is likely Poisson distributed. A Poisson exact test can then be performed to test the null hypothesis that the rate of mutation of each type is equal in the two types of chromatin.

The extent of this relationship can be further investigated, as the accessibility of the chromatin regions takes a numeric value. For each site, the normalized ATAC-seq coverage at that site and whether or not that contained a mutation can be regressed using a logistic regression. This type of regression is used when the dependent variable is binary; here, whether a mutation occured or not. This will test whether increasing accessibility is associated with an increased probability of experiencing a mutation. Additionally, this will estimate parameters that can be compared between the normal and cancer-burdened samples to determine how this relationship changes in diseased individuals.

Another method of comparing how the relationship between chromatin accessibility and mutation differs in tumors is by an analysis of the distribution of chromatin accessibility at mutated sites in both groups. This distribution can be calculated for each group then compared with a 2-sample Anderson Darling test. This tests the null hypothesis that the two samples were taken from the same distribution. It is non-parametric and does not assume an underlying distribution. This test was chosen over alternative tests because it does not penalize differences in the tails as heavily as other distributions. Since any site not determined to be significantly enriched will have an accessibility of 0, there may be an abundance of data on the left tail of the distribution. If this bias is found to be too extreme, the analysis can be done using only sites that are found to be in open regions.

At the conclusion of these analyses, the extent of the relationship between chromatin accessibility and mutation rate---and the way this relationship changes in cancer---will be elucidated.

\subsubsection{Anticipated results, potential pitfalls and alternative approaches}
\paragraph{Anticipated Results.}
As others have found in similar experiments, I expect to find that open chromatin regions correspond to an increased insertion/deletion mutation rate. Additionally, I expect that an increase in chromatin accessibility as assayed by ATAC-seq corresponds to an increase in mutation rate. Controlling for an increase in global mutation rate expected in tumor cells, I expect the relationship between chromatin accessibility and mutation rate to be even stronger in cancer, with only slight increases in accessibility resulting in large increases of mutation rate.

\paragraph{Pitfalls and Alternative Approaches.}
% To sensitively detect mutations as accurately as possible, PCR-free sequencing library preparations will be used. However, the amount of source cells may be too low to get sufficient DNA for mutation detection. After DNA extraction, if the concentration of DNA is observed to be too low for Illumina and Nanopore sequencing, PCR will be used during library preparation. This will increase the false positive rate and therefore increase the amount of effort needed to validate mutations using Sanger sequencing, but will provide enough data to perform the analysis.
Though the proposed method attempts to achieve a good balance between sufficient input material and reducing sample heterogeneity, the samples may be too heterogeneous to produce significant ATAC-seq signals or mutation calls. If this is the case, rather than sequencing the entire intestinal epithelium, only the largest tumor of the mouse will be excised and sequenced. As each of these tumors is clonal, there should be a significant reduction in heterogeneity. If this is still insufficient, single-cell extraction and sequencing methods will be used in place of those described above.

\subsection{Timeline}

Work on Aim 1 will take place for the first nine months. Work on Aim 2 will begin halfway through the first year and conclude at the end of the second year.

%TODO: timeline fig?

\clearpage

\printbibliography

\end{document}